src/market/market_utils.cairo

```diff
@@ -538,3 +546,162 @@ fn is_pnl_factor_exceeded(
     // TODO
     (true, 0, 0)
 }
+
+
+/// Validate that the specified market exists and is enabled
+/// # Arguments
+/// * `data_store` - The `DataStore` contract dispatcher.
+/// * `market` - The address of the market
+fn validate_enable_market(data_store: IDataStoreSafeDispatcher, market: Market) {
+    assert(market.market_token.is_non_zero(), 'EmptyMarket');
+    let is_market_disabled = data_store
+        .get_bool(keys::is_market_disabled_key(market.market_token))
+        .expect('validate_enable_market::result')
+        .expect('validate_enable_market::bool');
+    assert(is_market_disabled, 'DisabledMarket');
```

Comment:
I think it's assert(!is_market_disabled, 'DisabledMarket');

Reply:
Good catch